ext/ip_extraction.c

```diff
@@ -418,11 +458,11 @@ static bool dd_is_private_v6(const struct in6_addr *addr) {
     static const struct {
         union {
             struct in6_addr base;
-            uint64_t base_i[2];
+            unsigned __int128 base_i;
```

Comment:
I believe @bwoebi had some concerns about the `__int128`:

https://github.com/DataDog/dd-trace-php/pull/1621#discussion_r903664382